crates/bevy_winit/src/winit_config.rs

```diff
@@ -12,4 +14,76 @@ pub struct WinitConfig {
     /// `openbsd`. If set to true on an unsupported platform
     /// [run](bevy_app::App::run) will panic.
     pub return_from_run: bool,
+    /// Configures how the winit event loop updates while the window is focused.
+    pub focused_mode: UpdateMode,
+    /// Configures how the winit event loop updates while the window is *not* focused.
+    pub unfocused_mode: UpdateMode,
+}
+impl WinitConfig {
+    /// Configure winit with common settings for a game.
+    pub fn game() -> Self {
+        WinitConfig::default()
+    }
+    /// Configure winit with common settings for a desktop application.
+    pub fn desktop_app() -> Self {
+        WinitConfig {
+            focused_mode: UpdateMode::Reactive {
+                max_wait: Duration::from_secs(5),
+            },
+            unfocused_mode: UpdateMode::ReactiveLowPower {
+                max_wait: Duration::from_secs(60),
+            },
+            ..Default::default()
+        }
+    }
+    pub fn update_mode(&self, focused: bool) -> &UpdateMode {
+        match focused {
+            true => &self.focused_mode,
+            false => &self.unfocused_mode,
+        }
+    }
+}
+impl Default for WinitConfig {
+    fn default() -> Self {
+        WinitConfig {
+            return_from_run: false,
+            focused_mode: UpdateMode::Continuous,
+            unfocused_mode: UpdateMode::ReactiveLowPower {
+                max_wait: Duration::from_millis(100),
+            },
+        }
+    }
+}
+
+/// Configure how the winit event loop should update.
+#[derive(Debug)]
+pub enum UpdateMode {
+    /// The event loop will update continuously, running as fast as possible.
+    Continuous,
+    /// The event loop will only update if there is a winit event, a redraw is requested, or the
+    /// maximum wait time has elapsed.
+    ///
+    /// ## Note
+    ///
+    /// Once the app has executed all bevy systems and reaches the end of the event loop, there is
+    /// no way to force the app wake and update again, unless a winit event is received or the time
```

Comment:
Maybe people don't know what a winit event is, so perhaps an example like an input event or whatever is correct?